src/main/java/com/powsybl/openloadflow/util/MatrixUtil.java

```diff
@@ -11,11 +11,13 @@
 import com.powsybl.math.matrix.DenseMatrix;
 
 /**
- * TODO: move theses features to core.
+ * Do not use this class : all these features have been moved to powsybl.math.matrix.DenseMatrix
```

Comment:
Hi, the goal was to avoid breaking change for the (very unlikely) case of users using these methods out of OLF. It may be removed in future releases.

Reply:
but now this code is completely uncovered / untested. This would just leave technical debt in OLF.
Are there any known user using it ? And even if there are they should move to the core implementation.

Reply:
Ok you are right, no there is no known usage of it. I will delete this class then, it is simpler.